src/runtime_src/core/pcie/linux/pcidrv.h

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+
+#ifndef _XCL_PCIDRV_H_
+#define _XCL_PCIDRV_H_
+
+#include "pcidev.h"
```

Comment:
Looks like a circular dependency here.  pcidrv.h -> pcidev.h -> pcidrv.h

Reply:
Right, the forward declaration should be enough. I might have forgot to remove the included header after I added it. Will fix.